ui/README.md

```diff
@@ -0,0 +1,24 @@
+# clustogram
```

Comment:
I highly recommend sticking to some form of [my README template](https://github.com/allgreed/templates/blob/master/README.md_template), however I'm not having super strong opinion about this. Just please make sure that relevant `Makefile` targets work. It's easiest to achieve the make <-> README parity by simply having README redirect to make, hence my suggestion. But as long as Make works as expected you do you ;)

Reply:
I removed README from ui folder. I think it's not necessary if README in root directory exists already.